modules/tls-profiles-kubernetes-configuring.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * security/tls-profiles.adoc
+
+[id="tls-profiles-kubernetes-configuring_{context}"]
+= Configuring the TLS security profile for the Kubernetes control plane
+
+To configure a TLS security profile for the Kubernetes control components (Kubernetes API server, the Kubernetes scheduler, and the Kubernetes controller manager), edit the `APIServer` custom resource (CR).  If a TLS profile is not configured, the default TLS profile is `Intermediate`. The Kubernetes control plane consists of the Kubernetes API server, the Kubernetes scheduler, and the Kubernetes controller manager.
```

Comment:
"for the Kubernetes control *plane* components"

Reply:
I believe this configuration is also reflected in the openshift control plane components, like the openshift- and oauth-apiservers and the oauth-server. Are these addressed elsewhere?